frontend/components/Cell.js

```diff
@@ -214,3 +214,20 @@ export const Cell = ({
         </pluto-cell>
     `
 }
+
+export const MemoCell = (props) => {
```

Comment:
Quick question: why are you depending on _some_ props and not all?
Even if they are stable _now_ I'd still add it